docs/src/components/Layout/GlobalNav/GlobalNav.module.scss

```diff
@@ -22,11 +22,14 @@
 
   // Top level navbar element, create all of the variables that will be used throughout the unified navbar
   --navbar-border-color: var(--amplify-colors-neutral-80);
+  --navbar-padding-inline: var(--amplify-space-large);
   --navbar-current-property-color: #ff9900;
   --secondary-nav-hover-color: black;
```

Comment:
Where are these colors from? Maybe add a variable or at least some reference so that we can make it consistant in the future?

Same for line3-6

Reply:
I think these colors came from Danny and our designer. What kind of reference do you want to see, like me adding some extra comments about them or giving them a variable name?

Reply:
Do we have a design guideline or a color reference table, etc?